src/vs/workbench/contrib/debug/browser/media/repl.css

```diff
@@ -59,10 +59,6 @@
 	max-width: 150px;
 }
 
```

Comment:
This line should not be removed. This controls the looks of the cursor inside the debug console tree and is unrelated to this PR.

Reply:
@isidorn That is the entire point of this PR, to change how the mouse cursor looks like on mac, but it goes even outside of the editor and adopts the custom cursor that the editor had also in the repl and in the comments thread.

Reply:
Aha, I see that the class is set on the repl-tree.
I misunderstood the PR that it changes cursor only in the editor and all the editor instances like the repl input.
So I take my original comment back.